apps/zotonic_mod_cookie_consent/priv/templates/_media_cookie_consent.tpl

```diff
@@ -0,0 +1,29 @@
+{# Wrapper for HTML that needs cookie consent #}
+
+{% with consent|default:"all" as consent %}
+{% with viewer_options.mediaclass|default:"cookie-consent-preview" as mediaclass %}
+<div class="cookie-consent-preview do_cookie_consent mediaclass-{{ mediaclass }}" data-cookie-consent="{{ consent }}">
+    {% image id.depiction
+             mediaclass=mediaclass
+             alt=_"Media preview"
+             title=_"Click to change cookie settings"
+    %}
+    <p>
+        {_ Please consent to cookies to display external content. _}
+        <a href="#cookie-consent" id="{{ #consent }}" class="cookie-consent-change">{_ Change cookie settings _}</a>
+    </p>
+
+    
+    {% javascript %}
+    alert("bliep");
```

Comment:
@mworrell i can't seem to let this alert show itself, do you have a hint? Maybe because this template is inserted by the mod_cookie_consent.erl?

Reply:
Indeed, the HTML is used, the actions are discarded.
I have added comments to make that clear.